packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -34,6 +45,26 @@ export default class WebpackConfigGenerator {
 
   resolveConfig(config: Configuration | string): Configuration {
     if (typeof config === 'string') {
+      const ext = path.extname(config);
+      const interpreted = Object.keys(interpret.jsVariants).find((variant) => variant === ext);
+      if (interpreted) {
+        try {
+          rechoir.prepare(interpret.extensions, config);
+        } catch (error) {
+          if (isRechoirError(error)) {
+            const { message, failures } = error;
+            const logLines = [
+              `Unable to load webpack config '${config}'`,
+              message,
+              ...failures.map(({ error }) => error.message),
+              `Please install one of them`,
+            ];
+            logLines.forEach((line) => console.error(chalk.red(line)));
+            throw new Error(`Failed to load webpack config for ${config}`);
```

Comment:
This is done to prevent tests from encountering the `process.exit`. The only downside here is that combined with `async-ora` the output looks a little odd

![image](https://user-images.githubusercontent.com/44379112/156254414-69f38133-f7b0-4371-92f1-1fe89b710168.png)